orderbook/openapi.yml

```diff
@@ -880,18 +880,22 @@ components:
             partiallyFillable:
               description: Is this a fill-or-kill order or a partially fillable order?
               type: boolean
+              default: false
```

Comment:
Why default? I prefer making this explicit so that users have to set it because it can help prevent issues in the future.

Reply:
I also see a few `#[serde(default)]` below (added in previous PR) and have the same question there.

Reply:
I will remove it here and swing back to the other one if you can tell me a bit more specifically which you are referring to.

Reply:
Are you referring to these two fields?

```
            sellTokenBalance:
              $ref: "#/components/schemas/SellTokenSource"
              default: "erc20"
            buyTokenBalance:
              $ref: "#/components/schemas/BuyTokenDestination"
              default: "erc20"
```

Reply:
I feel like none of the order data should have defaults.

Reply:
All good, I believe these are the only two that remain. Will remove the defaults here.

Reply:
https://github.com/gnosis/gp-v2-services/pull/1189